network-api/networkapi/wagtailpages/templatetags/mini_site_tags.py

```diff
@@ -23,16 +24,21 @@ def mini_site_horizontal_nav(context, page):
 # Render a page's CTA (petition, signup, etc.)
 @register.inclusion_tag("wagtailpages/tags/cta.html", takes_context=True)
 def cta(context, page):
-    cta = {
-        "page": page,
-        "cta": None,
-        "cta_type": None,
-        "lang": context["request"].LANGUAGE_CODE,
-        "source_url": context["request"].build_absolute_uri(),
-        "thank_you_url": context["request"].build_absolute_uri(context["request"].path + "?thank_you=true"),
-        "show_formassembly_thank_you": context["request"].GET.get("thank_you") == "true",
-        "csp_nonce": context["request"].csp_nonce,
-    }
+
+    def generate_thank_you_url(url):
+        """
+        Generate a thank you page URL by grabbing the current url and appending thank_you=true
+        """
+
+        parsed_url = urlparse(url)
```

Comment:
Is there a simpler implementation for this? I was thinking to just test on the existence of `?` and append `thank_you=true` accordingly lol